extracts_v6.go

```diff
@@ -659,14 +666,186 @@ func extractEspionageReportMessageIDsFromDocV6(doc *goquery.Document) ([]Espiona
+						report.IsBandit = true
+					} else if banditstarlord.HasClass("rank_starlord1") || banditstarlord.HasClass("rank_starlord2") || banditstarlord.HasClass("rank_starlord3") {
+						report.IsStarlord = true
+					}
+				}
+
+				// IsInactive, IsLongInactive
+				report.IsInactive = false
+				report.IsLongInactive = false
+
+				if s.Find("span.status_abbr_longinactive").Size() > 0 {
+					report.IsInactive = true
+					report.IsLongInactive = true
+				} else if s.Find("span.status_abbr_inactive").Size() > 0 {
+					report.IsInactive = true
+				}
+
+				// HasFleet, FleetValue
+				report.HasFleet = false
+				report.FleetValue = -1
+
+				fleet := s.Find("div.compacting").Last().Find("span").Eq(0).AttrOr("title", "-1")
+
+				if strings.Contains(fleet, ":") {
+					fleet = strings.Split(fleet, ":")[1]
+					fleet = strings.TrimSpace(fleet)
+					report.FleetValue = ParseInt(fleet)
+				}
+
+				if report.FleetValue > -1 {
+					report.HasFleet = true
+				}
+
+				// HasDefenses,  DefenseValue
+				report.HasDefenses = false
+				report.DefenseValue = -1
+
+				defense := s.Find("div.compacting").Last().Find("span").Eq(1).AttrOr("title", "-1")
+				report.DefenseValue = ParseInt(defense)
+
+				if report.DefenseValue > -1 {
+					report.HasDefenses = true
+				}
+
+				// Metal, Crystal, Deuterium
+				log.SetFlags(log.LstdFlags | log.Lshortfile)
+
+				// res := s.Find("div.compacting").Eq(2).Find("span")
+				res := s.Find("span.resspan").Parent()
+
+				metal := res.Find("span.resspan").Eq(0).Text()
+				crystal := res.Find("span.resspan").Eq(1).Text()
+				deuterium := res.Find("span.resspan").Eq(2).Text()
+
+				log.Print("preParseres = ", metal, crystal, deuterium)
+
+				report.Metal = parseRes(metal)
+				report.Crystal = parseRes(crystal)
+				report.Deuterium = parseRes(deuterium)
+
+				log.Print(report.Metal, report.Crystal, report.Deuterium)
+
+				// APIKey
+				apikey, _ := s.Find("span.icon_apikey").Attr("title")
+				blaat, _ := goquery.NewDocumentFromReader(strings.NewReader(apikey))
+				snarfzonk, _ := blaat.Find("input").First().Attr("value")
+
+				report.APIKey = snarfzonk
+
+				// CreatedAt
+				// <span class="msg_date fright">09.02.2019 23:25:56</span>
+				msgDate, _ := time.Parse("02.01.2006 15:04:05", s.Find("span.msg_date").Text())
+				report.CreatedAt = msgDate
+
+//log.Fatal("--end--")
+				msgs = append(msgs, report)
 			}
 		}
 	})
 	return msgs, nbPage
 }
 
+func parseRes(input string) (res int64) {
+	// Metaal: 2,4M
+	// Kristal: 3,6M
+	// Deuterium: 4,8M
+	// Deuterium: 140.000
+
+	//log.Print("Original input: ", input)
+
+	if strings.Contains(input, ":") {
+		input = strings.Split(input, ":")[1]
+	}
+
+	if strings.Contains(input, ",") {
+		input = strings.Replace(input, ",", ".", 1)
+	}
+
+	if strings.Contains(input, "Miljard") { // Todo: Billion in English? Or just return the value as string instead of int64
+		//log.Print(2.1 * 1e9)
```

Comment:
Can you remove all the debugging logs and comments ?
By the way, you can know the type of any variable by pressing `Ctrl+Shift+P` in GoLand.

Reply:
I've removed them